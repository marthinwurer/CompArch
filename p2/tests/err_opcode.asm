;
; tests erroneous opcodes
;
; uses instructions HALT
;
; uses no addressing modes
;
	.globl	start
;
start:
	.word	1
	halt
;
	.end	start
