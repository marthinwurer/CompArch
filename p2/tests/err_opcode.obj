000 2 01 00
002 2 00 00
000
