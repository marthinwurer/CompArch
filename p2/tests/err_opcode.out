CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 0
000000:  PS[00]  000001  ????

Machine Halted - opcode error

Simulated time 5 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
